### PR TITLE
launch example of map_server

### PR DESCRIPTION
**Pull request by luisCFsantos**
I added an easy example to use map server with launch file that already triggers the lifecycle

---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | (add tickets here #1) |
| Primary OS tested on | (Ubuntu, MacOS, Windows) |
| Robotic platform tested on | (Steve's Robot, gazebo simulation of Tally, hardware turtlebot) |

---

## Description of documentation updates required from your changes

---

#### For Maintainers: 
- [ ] Check that any new parameters added are updated in navigation.ros.org
- [ ] Check that any significant change is added to the migration guide
- [ ] Check that any new features **OR** changes to existing behaviors are reflected in the tuning guide
- [ ] Check that any new functions have Doxygen added
- [ ] Check that any new features have test coverage
- [ ] Check that any new plugins is added to the plugins page
- [ ] If BT Node, Additionally: add to BT's XML index of nodes for groot, BT package's readme table, and BT library lists

### CODE REVIEW

File: nav2_map_server/README.md [hunk 1]
```diff
@@ -100,6 +100,22 @@ Then, one would invoke this process with the params file that contains the param
 $ process_with_multiple_map_servers __params:=combined_params.yaml
 ```
 
+**Attention** : map_server is now a lifecycle node and needs to be transitioned to the active state. There are several ways you can trigger that state transition. The easiest way from the command line is:
```
Comment (SteveMacenski) [inline nav2_map_server/README.md:103]: Remove this, this is a well documented item

Comment (SteveMacenski) [reply nav2_map_server/README.md:103]: With that gone, I think the flow of this documentation section changes.

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: nav2_map_server/README.md (hunk 1 above)
Comment (luisCFsantos) [reply nav2_map_server/README.md:103]: I had to read another issue to discover this. It does not seem to be documented in the nav2_map_server package. But if you insist, I will remove it.

Comment (SteveMacenski) [reply nav2_map_server/README.md:103]: You should still keep the ros2 run line for calling the lifecycle manager util for transitioning up the server, so I think that will be abundantly clear from context just below this line that its a lifecycle server

**Comment by SteveMacenski** [comment]
Any update?

**Comment by SteveMacenski** [comment]
This work should also target `main` branch, where all new development goes

**Comment by SteveMacenski** [comment]
Any update? its been quite a while

**Comment by SteveMacenski** [comment]
Closing due to lack of response, happy to have you reopen it once you handle the review comments.

**Pull request closed without merging**